tools/src/wyvern/stdlib/support/AST.java

```diff
@@ -301,14 +304,61 @@ public IExpr parseExpression(String input, GenContext ctx) throws ParseException
         }
     }
 
+    public TypedAST parseGeneratedModule(String input) throws ParseException {
+        InterpreterState state = getLocalThreadInterpreter();
+        GenContext genContext = Globals.getGenContext(state);
+        try {
+            TypedAST typAST = TestUtil.getNewAST(input.trim() + "\n", "TSL Parse");
+            if (typAST instanceof wyvern.tools.typedAST.core.declarations.ModuleDeclaration) {
+                return (wyvern.tools.typedAST.core.declarations.ModuleDeclaration) typAST;
+            } else {
+                throw new RuntimeException("Unexpected parsing input is not a module declaration: \"" + input + "\"");
+            }
+        } catch (ParseException e) {
+            System.err.println("Error when running parseExpression on input \"" + input + "\"");
+            throw e;
+        }
+    }
+
+    public Object parseExpressionNoContext(String input) throws ParseException {
+        InterpreterState state = getLocalThreadInterpreter();
+        GenContext genContext = Globals.getGenContext(state);
+        try {
+            TypedAST typAST = TestUtil.getNewAST(input.trim() + "\n", "TSL Parse");
```

Comment:
Please change the second parameter of the getNewAST() function to something more appropriate.